Sources/SimplyCoreAudio/Public/Enums/Scope.swift

```diff
@@ -57,6 +61,7 @@ extension Scope {
         case kAudioObjectPropertyScopeOutput: return .output
         case kAudioObjectPropertyScopePlayThrough: return .playthrough
         case kAudioObjectPropertyElementMaster: return .master
```

Comment:
could prefer for this to return `.main`; as a runtime concern, don't know it would matter; but there is a break in ~invertibility between this and the above